internal/proxy/oauthproxy.go

```diff
@@ -167,7 +154,21 @@ func (t *upstreamTransport) RoundTrip(req *http.Request) (*http.Response, error)
 // It adds an X-Forwarded-Host header that is the request's host.
 func NewReverseProxy(to *url.URL, config *UpstreamConfig) *httputil.ReverseProxy {
 	proxy := httputil.NewSingleHostReverseProxy(to)
-	proxy.Transport = &upstreamTransport{InsecureSkipVerify: config.TLSSkipVerify}
+	proxy.Transport = &upstreamTransport{
```

Comment:
nit: we can probably create a function that initializes the `upstreamTransport{}`, something like `NewUpstreamTransport(insecureSkipVerify bool)` so that we're not repeating the config code twice?

Reply:
Yeah, I like that solution better.